L1Trigger/L1THGCalUtilities/plugins/ntuples/HGCalTriggerNtupleHGCMulticlusters.cc

```diff
@@ -162,11 +162,14 @@ void HGCalTriggerNtupleHGCMulticlusters::fill(const edm::Event& e, const edm::Ev
     cl3d_bdteg_.emplace_back(id_->value(*cl3d_itr));
     cl3d_quality_.emplace_back(cl3d_itr->hwQual());
     if (fill_interpretation_info_) {
-      std::vector<float> iPts;
-      std::vector<float> iEnergies;
-      for (auto interp : cl3d_itr->energyInterpretations()) {
-        iPts.emplace_back(cl3d_itr->iPt(interp));
-        iEnergies.emplace_back(cl3d_itr->iEnergy(interp));
+      std::vector<float> iPts(
+          std::distance(cl3d_itr->energyInterpretations().first, cl3d_itr->energyInterpretations().second));
```

Comment:
The number of interpretations could be implemented in an additional method of `HGCalMultiCluster`. It would probably be clearer.